packages/cloud_firestore/ios/Classes/CloudFirestorePlugin.m

```diff
@@ -39,7 +39,13 @@
 }
 
 static FIRQuery *getQuery(NSDictionary *arguments) {
-  FIRQuery *query = [getFirestore(arguments) collectionWithPath:arguments[@"path"]];
+  BOOL isCollectionGroup = arguments[@"collectionGroup"];
```

Comment:
I think you are probably implicitly treating an `NSNumber *` as a `BOOL` here. It doesn't really matter since `collectionGroup` is always true if set, but it's a hazard that could cause bugs in the future. Consider making it an `NSNumber *` and use `.boolValue`.